backend/app/services/share/knowledge_share_service.py

```
@@ -274,12 +277,12 @@ def can_manage_permissions(
         user_id: int,
     ) -> bool:
         """Check if user can manage permissions for a knowledge base."""
-        has_access, level, is_creator = self.get_user_kb_permission(
+        has_access, role, level, is_creator = self.get_user_kb_permission(
```

Ruff (0.15.2)

[warning] Unpacked variable `level` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)